README.md

```diff
@@ -50,5 +50,5 @@ Find out more on the <a href="https://layer5.io/community">Layer5 community</a>.
 ✔️ <em><strong>Discuss</strong></em> in the <a href="https://discuss.layer5.io">Community Forum</a>.<br />
 </p>
 <p align="center">
-<i>Not sure where to start?</i> Grab an open issue with the <a href="https://github.com/issues?q=is%3Aopen+is%3Aissue+archived%3Afalse+org%3Alayer5io+org%3Ameshery+org%3Aservice-mesh-performance+org%3Aservice-mesh-patterns+label%3A%22help+wanted%22+">help-wanted label</a>.
+<i>Not sure where to start?</i> Grab an open issue with the <a href="https://github.com/meshery/meshkit/labels/help%20wanted">help-wanted label</a> or look at our  <a href="https://github.com/meshery/meshkit/labels/good%20first%20issue">good-first issues.
```

Comment:
@Ghat0tkach Please revert this. This global search is intentional.

Reply:
Oh , I apologize i didnt knew about it